### PR TITLE
Feature lwdev 6492 all client wallets

### PR DESCRIPTION
**Pull request by OttoVT**

### CODE REVIEW

File: src/Lykke.Service.BlockchainWallets.AzureRepositories/WalletRepository.cs [hunk 1]
```diff
@@ -157,5 +200,34 @@ internal async Task<(IEnumerable<IWallet> wallets, string continuationToken)> Ge
...
+            var indexes = await GetForClientIndixesAsync(clientId, take, continuationToken);
+            var keys = indexes.wallets.Select(x => Tuple.Create<string, string>(x.walletPartitionKey, x.walletRowKey));
+            var wallets = await _walletsTable.GetDataAsync(keys, take);
+
+            return (wallets, indexes.continuationToken);
+        }
+
+        // NB! This method should be used only by conversion utility or in similar cases.
+        internal async Task<(IEnumerable<(string walletPartitionKey, string walletRowKey)> wallets, string continuationToken)> 
+            GetForClientIndixesAsync(Guid clientId, int take, string continuationToken)
```
Comment (PaterSantyago) [inline src/Lykke.Service.BlockchainWallets.AzureRepositories/WalletRepository.cs]: Indexes on Indices, not Indixes :-)

File: src/Lykke.Service.BlockchainWallets/Controllers/WalletsController.cs [hunk 2]
```diff
@@ -3,16 +3,18 @@
...
 using Lykke.Service.BlockchainWallets.Models;
 using Microsoft.AspNetCore.Mvc;
 
 namespace Lykke.Service.BlockchainWallets.Controllers
 {
-    [Route("api/wallets/{integrationLayerId}/{integrationLayerAssetId}")]
+    [Route("api/wallets")]
     public class WalletsController : Controller
     {
+        public const string _routeSuffix = "{integrationLayerId}/{integrationLayerAssetId}";
```
Comment (PaterSantyago) [inline src/Lykke.Service.BlockchainWallets/Controllers/WalletsController.cs]: Why this constant is public? Also, suggested naming for this constant is `RouteSuffix`

File: client/Lykke.Service.BlockchainWallets.Client/BlockchainWalletsClient.cs [hunk 3]
```diff
@@ -194,6 +197,46 @@ private static void ValidateInputParameters(string integrationLayerId, string as
...
+            {
+                throw new ArgumentException(nameof(clientId));
+            }
+
+            string continuationToken = null;
+            List<ClientWalletResponse> wallets = new List<ClientWalletResponse>(batchSize);
+
+            do
+            {
+                var response = await _apiRunner.RunWithRetriesAsync(() =>
```
[the remark this answers is not in the document]
Comment (OttoVT) [reply client/Lykke.Service.BlockchainWallets.Client/BlockchainWalletsClient.cs]: fixed

File: client/Lykke.Service.BlockchainWallets.Client/BlockchainWalletsClient.cs [hunk 4]
```diff
@@ -194,6 +197,46 @@ private static void ValidateInputParameters(string integrationLayerId, string as
...
+
+        public async Task<IEnumerable<ClientWalletResponse>> TryGetAllClientWalletsAsync(Guid clientId, int batchSize = 50)
+        {
+            if (clientId == Guid.Empty)
+            {
+                throw new ArgumentException(nameof(clientId));
+            }
+
+            string continuationToken = null;
+            List<ClientWalletResponse> wallets = new List<ClientWalletResponse>(batchSize);
```
[the remark this answers is not in the document]
Comment (OttoVT) [reply client/Lykke.Service.BlockchainWallets.Client/BlockchainWalletsClient.cs]: fixed

File: src/Lykke.Service.BlockchainWallets.AzureRepositories/WalletRepository.cs (hunk 1 above)
Comment (OttoVT) [reply src/Lykke.Service.BlockchainWallets.AzureRepositories/WalletRepository.cs]: Oops, fixed

File: src/Lykke.Service.BlockchainWallets/Controllers/WalletsController.cs (hunk 2 above)
Comment (OttoVT) [reply src/Lykke.Service.BlockchainWallets/Controllers/WalletsController.cs]: fixed also

**Pull request merged**